sprint0/sprint0/Items/Bomb.cs

```diff
@@ -101,9 +101,14 @@ private bool Alive()
             return false;
         }
 
+        public void Perish()
```

Comment:
I am confused what this does, can you add comments. Does it do what lines 125 to 130 do? Or like should we move those there and then call perish to make update method prettier?